src/views/Tournament/Tournament.tsx

```diff
@@ -242,15 +255,17 @@ export function Tournament(): JSX.Element {
             : null;
 
     const raw_selected_round =
-        typeof selected_round_idx === "number" && rounds && rounds.length > selected_round_idx
+        typeof selected_round_idx === "number" &&
+        raw_rounds &&
+        raw_rounds.length > selected_round_idx
             ? raw_rounds[selected_round_idx]
             : null;
 
     // this is so anoek (user id 1) can quickly test tournaments
     React.useEffect(() => {
         if (user.id === 1 && tournament_id === 0) {
             setTournament({
-                ...tournament,
+                ...default_tournament,
```

Comment:
@anoek, for obvious reasons, I haven't been able to test this part of the code. I also suspect it blocks "Clone Tournament" from working for you.

Two options to consider:

- Delete this block entirely now that we have "Clone Tournament"? It seems like the only thing you're gaining vs. clicking "Clone Tournament" is not having to set `time_start`.
- Lift this up, changing how `default_tournament` gets set based on `user.id === 1` instead of modifying the tournament later via `useEffect`. This would be easier to reason about.

Reply:
I implemented and pushed the second option in 80e5e8f5. But it might be nice to delete this entirely, or limit it just to a `time_start` override, if you can use "Clone Tournament" instead.